src/hero/_base.scss

```diff
@@ -7,6 +7,7 @@
 .hero {
   display: flex;
   flex-direction: column;
+  align-items: center;
   justify-content: center;
   min-height: 300px;
```

Comment:
It's going to be better for us if we avoid pixel values whenever possible. When we want to set height, let's think about it in terms of viewport units: what is the proportion/ratio of the screen we want the element to take up?